build/packaging/suseconnect-ng.changes

```diff
@@ -7,6 +7,9 @@ Thu Apr 25 15:39:00 UTC 2024 - Felix Schnizlein <fschnizlein@suse.com>
   * Add --gpg-auto-import-keys flag before action in zypper command (bsc#1219004)
   * Include /etc/products.d in directories whose content are backed
     up and restored if a zypper-migration rollback happens. (bsc#1219004)
+  * Add the ability to upload the system uptime logs, produced by the
+    suse-uptime-tracker daemon, to SCC/RMT as part of daily heartbeat report.
```

Comment:
Again: `heartbeat` -> `keepalive`. Moreover, I'd remove the `daily` because that's not guaranteed since customers can always disable the daily timer and that's ok.

Reply:
Done